src/Text/Parsing/Parser.purs

```diff
@@ -161,8 +161,6 @@ instance plusParserT :: Monad m => Plus (ParserT s m) where
 
 instance alternativeParserT :: Monad m => Alternative (ParserT s m)
 
-instance monadZeroParserT :: Monad m => MonadZero (ParserT s m)
```

Comment:
I thought we weren't removing these until v0.15.0 comes out?

Reply:
Oh? Thanks @JordanMartinez good catch.

Reply:
I can't find a reference for this, can you post a link please @JordanMartinez ?

Reply:
See https://github.com/purescript/purescript-lists/issues/176

We have a couple of `MonadZero` issues like that scattered throughout the core and contrib libs.

Reply:
There’s a strict requirement that we don’t remove the warning in core libraries, and we’ve largely stuck to that in non-core libraries too. But there’s not a _requirement_ that we keep these in place other than the core org  (I’ve removed it elsewhere when making breaking changes 😅)